planning/behavior_path_start_planner_module/src/start_planner_module.cpp

```diff
@@ -720,27 +720,36 @@ PathWithLaneId StartPlannerModule::extractCollisionCheckSection(
 
   const double collision_check_distance_from_end = collision_check_distances.at(planner_type);
 
-  PathWithLaneId combined_path;
+  PathWithLaneId full_path;
   for (const auto & partial_path : path.partial_paths) {
-    combined_path.points.insert(
-      combined_path.points.end(), partial_path.points.begin(), partial_path.points.end());
+    full_path.points.insert(
+      full_path.points.end(), partial_path.points.begin(), partial_path.points.end());
   }
-  // calculate collision check end idx
-  const size_t collision_check_end_idx = std::invoke([&]() {
-    const auto collision_check_end_pose = motion_utils::calcLongitudinalOffsetPose(
-      combined_path.points, path.end_pose.position, collision_check_distance_from_end);
 
-    if (collision_check_end_pose) {
-      return motion_utils::findNearestIndex(
-        combined_path.points, collision_check_end_pose->position);
-    } else {
-      return combined_path.points.size() - 1;
-    }
-  });
-  // remove the point behind of collision check end pose
-  combined_path.points.erase(
-    combined_path.points.begin() + collision_check_end_idx + 1, combined_path.points.end());
-  return combined_path;
```

Comment:
memo: 
this kept points before all points before collision_check_end_idx including points before start pose